curriculum/challenges/english/25-front-end-development/workshop-plant-nursery-catalog/6737a95d5b431860f96ca7bb.md

```diff
@@ -0,0 +1,113 @@
+---
+id: 6737a95d5b431860f96ca7bb
+title: Step 27
+challengeType: 0
+dashedName: step-27
+---
+
+# --description--
+
+Now try to add again the `ballerina` object to `catalogSet` inside your function. You'll see that nothing happens, because the object is already stored in your set and duplicate elements are not allowed in sets.
```

Comment:
```suggestion
Now try to add the `ballerina` object to your `catalogSet` inside your function a second time. You'll see that nothing happens, because the object is already stored in your set and duplicate elements are not allowed in sets.
```